### PR TITLE
deps(transport-interop): add `v0.53` release of `rust-libp2p`

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: transport-interop/versions.ts [hunk 1]
```diff
@@ -62,12 +62,28 @@ export const versions: Array<Version> = [
...
         transports: [{ name: "webtransport", onlyDial: true }],
         secureChannels: [],
         muxers: [],
     },
+    {
+        id: "rust-chromium-v0.53",
+        "transports": [
+            { "name": "webtransport", "onlyDial": true },
+            { "name": "webrtc-direct", "onlyDial": true },
+            { "name": "ws", "onlyDial": true }
```
Comment (mxinden) [inline transport-interop/versions.ts:82]: :tada:

**Review by mxinden — APPROVED** [review]
Thank you for following up here!

Feel free to merge here once ready!

**Comment by thomaseizinger** [comment]
> Feel free to merge here once ready!

Does this mean you are okay with bumping this to a non-released commit on `master` or should we wait until we make another point-release of `libp2p`?

**Comment by mxinden** [comment]
Non-released commit on `master` works for me.

**Pull request merged**